onetl/impl/generic_options.py

```diff
@@ -59,10 +59,19 @@ def parse(
 
     @root_validator(pre=True)
     def strip_prefixes(cls, values):
-        if hasattr(cls.Config, "strip_prefixes"):
-            for value in values.copy():
-                for prefix in cls.Config.strip_prefixes:
-                    cls._handle_prefix(prefix, value, values)
+        if not hasattr(cls.Config, "strip_prefixes"):
+            return values
+        for value in values.copy():
```

Comment:
```suggestion
        for key in list(values.keys()):
```

Перебираются не значения, а ключи. Также нет смысла копировать весь dict, достаточно списка ключей.